GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -67,7 +67,7 @@ protected override void OnRuntimeLoad(EventArgs e)
 
             _NO_TRANSLATE_To.Text = AppSettings.DefaultCloneDestinationPath;
 
-            if (url.IsNotNullOrWhitespace())
+            if (GitModule.PathIsUrl(url) && GitModule.IsValidGitWorkingDir(url))
```

Comment:
It can be an URL when you start GitExtensions with an URL in the CommandLine Arguments, Like the URL handler for github-windows:// does

    GitExtensions.exe github-windows://...

And the Git Clone Explorer command pass the current working directory to GitExtensions like this:

    GitExtensions clone c:\src\...

So url can contains an url or a path and for me it makes only sense to put that url into the From field, when git can clone from that location.